src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PAYMENT = 40;
+  const BIGGER_DISCONT = 7;
+  const SMALLER_DISCONT = 3;
+  let sum = days * PAYMENT;
+
+  if (days >= BIGGER_DISCONT) {
+    sum -= 50;
```

Comment:
The discount value '50' is hard-coded here. Use a constant with a descriptive name, such as 'LONG_TERM_DISCOUNT', to represent this value.